### PR TITLE
SQS Sample

### PR DESCRIPTION
**Pull request by MatejNedic**
## :loudspeaker: Type of change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
Implemented Sqs sample.

## :bulb: Motivation and Context
Currently there is no displayed way how sqs integration works. Now you can easily use cdk to launch resources and spring boot aplication that show cases basic use of integration.

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] I updated reference documentation to reflect the change
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

**Review by maciejwalkowiak — APPROVED** [review]
I think this example can be extended, show that we can deserialize JSON to objects, retrieve headers from messages etc. Potentially with multiple `@SqsListener` annotated methods. 

This all can be done in subsequent PRs.

File: spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cloud:
+    aws:
+        region:
+            static: eu-central-1
```
Comment (maciejwalkowiak) [inline spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml:4]: if we enforce region here, it should be also enforced in CDK. Or dropped in both places.

File: spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+        sqs:
```
Comment (maciejwalkowiak) [inline spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml:5]: There should be no need to explicitly set `sqs.enabled` to true, right?

File: .idea/misc.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+	<component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (maciejwalkowiak) [inline .idea/misc.xml:10]: this change should be reverted.

File: spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml (hunk 2 above)
Comment (MatejNedic) [reply spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml:5]: True will fix it got left

**Comment by maciejwalkowiak** [comment]
@MatejNedic could you please resolve conflicts?

**Comment by MatejNedic** [comment]
@maciejwalkowiak resolved

File: spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml (hunk 2 above)
Comment (maciejwalkowiak) [reply spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml:3]: This is still waiting to be removed ;-)

Comment (MatejNedic) [reply spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml]: Ups :)

**Pull request merged**